test/functional/wallet_listtransactions.py

```diff
@@ -284,6 +292,22 @@ def run_invalid_parameters_test(self):
         assert_raises_rpc_error(-8, "Negative count", self.nodes[0].listtransactions, count=-1)
         assert_raises_rpc_error(-8, "Negative from", self.nodes[0].listtransactions, skip=-1)
 
+    def test_op_return(self):
+
+        data = random_bytes(8)
+
+        tx = CTransaction()
+        tx.vout = [CTxOut(nValue=2, scriptPubKey=CScript([OP_RETURN, data]))]
```

Comment:
would be shorter to just call `createrawtransaction '[]' '[{"data":"aa"}]'`?

Reply:
Done in Done in https://github.com/bitcoin/bitcoin/pull/26349/commits/670395cfa65988ab09afd59854d0d64c0c1438ba. Thanks.